src/main/java/org/apache/mesos/protobuf/Devolver.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.mesos.protobuf;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import org.apache.mesos.v1.Protos;
+
+/**
+ * Utility class for devolving vX protos to v0.
+ */
+public class Devolver {
+    private static <T extends Message> T devolve(Message message, Message.Builder builder) {
```

Comment:
The functions `devolve` and `evolve` differ only in name.  Please keep code DRY.